pkg/otlp/internal/logsagentexporter/logs_exporter.go

```diff
@@ -52,13 +52,13 @@ func createConsumeLogsFunc(logger *zap.Logger, logSource *sources.LogSource, log
 					log := lsl.At(k)
 					ddLog := logsmapping.Transform(log, res, logger)
 
-					content, err := ddLog.MarshalJSON()
-					if err != nil {
-						logger.Error("Error parsing log: " + err.Error())
+					var tags []string
+					if ddTags := ddLog.GetDdtags(); ddTags == "" {
+						tags = []string{otelTag}
+					} else {
+						tags = append(strings.Split(ddTags, ","), otelTag)
 					}
-
-					tags := append(strings.Split(ddLog.GetDdtags(), ","), otelTag)
-					// TODO: remove tags in ddLog.Ddtags
+					ddLog.Ddtags = nil
```

Comment:
Add a comment that Ddtags are set in Origin.Tags instead.

Reply:
Does this need to be commented? 🤔 To me, the "tags" variable seems self-explanatory

Reply:
Readers then need to search for where `tags` are used